concordia/views.py

```diff
@@ -221,26 +221,23 @@ def post(self, *args, **kwargs):
             cmd = 'cp -r {0}/* {1}'.format('/concordia_images', collection_path)
             if os.WEXITSTATUS(os.system(cmd)) == 0:
                 os.system('rm -rf {0}'.format('/concordia_images/*'))
-            c = Collection.objects.create(title=name, slug=name.replace(" ", "-"), description=name)
-            count = 0
-            for root, dirs, files in os.walk(collection_path):
-                for filename in files:
-                    filename = os.path.join(root, filename)
-                    if True:
-                        count += 1
-                        title = '{0} asset {1}'.format(name, count)
-                        media_url = os.path.join(root, filename).replace(settings.MEDIA_ROOT, '')
+                c = Collection.objects.create(title=name, slug=name.replace(" ", "-"), description=name)
+                for root, dirs, files in os.walk(collection_path):
+                    for filename in files:
+                        file_path = os.path.join(root, filename)
+                        title = file_path.replace(collection_path + '/', '').split('/')[0]
+                        media_url = file_path.replace(settings.MEDIA_ROOT, '')
+                        sequence = filename.replace('.jpg', '')
                         Asset.objects.create(title=title,
-                                             slug=title.replace(" ", "-"),
+                                             slug=title + sequence,
                                              description="{0} description".format(title),
                                              media_url=media_url,
                                              media_type='IMG',
+                                             sequence=int(sequence),
```

Comment:
This should be converted at the time it's extracted from the filename and there should be a good user-visible error message instead of just bubbling a `ValueError` up. As with the rest of this code, it would also be good to know what the error handling strategy is — i.e. should this block be in a transaction?, what do we do with the files?, do we abort immediately as it currently does or just report and skip the problematic file?, etc.